src/metabase/cmd/endpoint_dox.clj

```diff
@@ -16,10 +16,16 @@
   {:servers [{:url         "http://localhost:3000/api"
               :description "Localhost"}]})
 
+(def ^:private scalar-info
+  "Info to include at the top of the page."
+  {:info {:title "Metabase API documentation"
+          :description (slurp (io/resource "openapi/api-intro.md"))}})
+
 (defn- openapi-object []
   (merge
    (metabase.api.common/openapi-object #'metabase.api.routes/routes)
-   scalar-config))
+   scalar-config
+   scalar-info))
```

Comment:
I would just include the `:info` key in `scalar-config` there's no real benefit from having two separate maps here they're both used in exactly this one place